tests/python/test_relational_vcfs.py

```diff
@@ -0,0 +1,90 @@
+import nose
+
+import tempfile
+
+
+
+def test_vcf_to_csv():
+    import vcf, csv
+    from common.relational_vcf import vcf_to_csv
```

Comment:
Nothing; don't recall why I put them there. Maybe to keep code specific to a tests in the test only? Regardless… moved them to the top.